docker/test/integration/features/kafka.feature

```diff
@@ -57,3 +57,179 @@ Feature: Sending data to using Kafka streaming platform using PublishKafka
 
     When both instances start up
     Then a flowfile with the content "test" is placed in the monitored directory in less than 60 seconds
+
+  Scenario: MiNiFi consumes data from a kafka topic
+    Given a ConsumeKafka processor set up in a "kafka-consumer-flow" flow
+    And a PutFile processor with the "Directory" property set to "/tmp/output" in the "kafka-consumer-flow" flow
+    And the "success" relationship of the ConsumeKafka processor is connected to the PutFile
+
+    And a kafka broker "broker" is set up in correspondence with the third-party kafka publisher
+
+    When all instances start up
+    And a message with content "some test message" is published to the "ConsumeKafkaTest" topic
+
+    Then at least one flowfile with the content "some test message" is placed in the monitored directory in less than 60 seconds
+
+  Scenario Outline: ConsumeKafka parses and uses kafka topics and topic name formats
+    Given a ConsumeKafka processor set up in a "kafka-consumer-flow" flow
+    And the "Topic Names" of the ConsumeKafka processor is set to "<topic names>"
+    And the "Topic Name Format" of the ConsumeKafka processor is set to "<topic name format>"
+    And a PutFile processor with the "Directory" property set to "/tmp/output" in the "kafka-consumer-flow" flow
+    And the "success" relationship of the ConsumeKafka processor is connected to the PutFile
+
+    And a kafka broker "broker" is set up in correspondence with the third-party kafka publisher
```

Comment:
The kafka option `allow.auto.create.topics` cannot work with regex patterns, and there is also no renegotiation per processor trigger on the topics list. Therefore one must manually set up the topics used beforehand.